ush/config_defaults.yaml

```diff
@@ -2811,63 +2107,27 @@ verification:
   OBS_NDAS_SFCorUPA_FN_TEMPLATE: 'prepbufr.ndas.{valid?fmt=%Y%m%d%H}'
   OBS_NDAS_SFCorUPA_FN_METPROC_TEMPLATE: '${OBS_NDAS_SFCorUPA_FN_TEMPLATE}.nc'
   #
-  # VX_LOCAL_MODULE_FN:
-  # Name (without extension) of the local module file for running the vx
-  # tasks in the workflow.
-  #
-  VX_LOCAL_MODULE_FN: 'run_vx'
-  #
-  # RUN_TASKS_METVX_DET:
-  # Flag that specifies whether to run deterministic verification.  If set
-  # to True, this will run deterministic vx on the post-processed forecast
-  # output.  This post-processed output may consist of a single forecast
-  # or an ensemble of foreasts, and it may be staged from previous runs of
-  # the SRW App or may be generated by running the TN_RUN_FCST task as part
-  # of the current SRW-App-generated experiment.
-  #
-  # RUN_TASKS_METVX_ENS:
-  # Flag that specifies whether to run ensemble verification.  The ensemble
-  # forecast output on which vx will be run may be staged or generated by
-  # running an ensemble of forecasts with the weather model as part of the
-  # current SRW-App-generated experiment.
-  #
-  RUN_TASKS_METVX_DET: False
-  RUN_TASKS_METVX_ENS: False
-  #
   # VX_FCST_MODEL_NAME:
   # String that specifies a descriptive name for the model being verified.
   # This is used in forming the names of the verification output files as
   # well as in the contents of those files.
   #
-  # VX_FIELDS:
-  # The fields or groups of fields on which to run verification.
-  #
-  # VX_APCP_ACCUMS_HH:
-  # The 2-digit accumulation periods (in units of hours) to consider for
-  # APCP (accumulated precipitation).  If VX_FIELDS contains "APCP", then
-  # VX_APCP_ACCUMS_HH must contain at least one element.  If not,
-  # VX_APCP_ACCUMS_HH will be ignored.
-  #
   VX_FCST_MODEL_NAME: '{{ nco.NET }}.{{ task_run_post.POST_OUTPUT_DOMAIN_NAME }}'
-  VX_FIELDS: [ "APCP", "REFC", "RETOP", "SFC", "UPA" ]
-  VX_APCP_ACCUMS_HH: [ "01", "03", "06", "24" ]
   #
-  # VX_FCST_INPUT_BASEDIR:
-  # Location of top-level directory containing forecast (but not obs) files
-  # that will be used as input into METplus for verification.  If not
-  # specified, this gets set to EXPTDIR.
+  # VX_FCST_INPUT_DIR:
+  # Directory template for the path containing forecast (but not obs) files
+  # that will be used as input into METplus for verification.
   #
   # VX_OUTPUT_BASEDIR:
   # Top-level directory in which METplus will place its output.
   #
-  VX_FCST_INPUT_BASEDIR: '{{ "$COMOUT/../.." if user.RUN_ENVIR == "nco" else ( workflow.EXPTDIR if ((workflow_switches.RUN_TASK_RUN_FCST and task_run_fcst.WRITE_DOPOST) or workflow_switches.RUN_TASK_RUN_POST) ) }}'
```

Comment:
@mkavulich The if-statement that's removed here was to make sure that this variable gets set to `EXPTDIR` only if UPP is run to generate forecast files.  Otherwise, it shouldn't be set to `EXPTDIR`.  I guess the task switches aren't available any more, so is this now done somewhere else, e.g. `setup.py`?